src/java/org/apache/cassandra/config/Config.java

```diff
@@ -1283,4 +1283,21 @@ public static void log(Config config)
     public double severity_during_decommission = 0;
 
     public StorageCompatibilityMode storage_compatibility_mode;
+
+    public enum CQLStartTime
+    {
+        REQUEST, // uses a timestamp that represent the start of processing of the request
+        QUEUE    // uses a timestamp that represents when the request was enqueued
+    }
+    public volatile CQLStartTime cql_start_time = CQLStartTime.REQUEST;
+
+    public boolean native_transport_throw_on_overload = false;
+    public double native_transport_queue_max_item_age_threshold = Double.MAX_VALUE;
```

Comment:
nit: I want to try to work the token "ratio" into the name of this one, but haven't been able to come up w/ something concrete :D